downstream/modules/automation-hub/proc-edit-namespace.adoc

```diff
@@ -4,14 +4,14 @@
 
 = Adding additional information and resources to a namespace
 
-You can add information and provide resources for your users to accompany collections included in the namespace. Add a logo, description and link users to your GitHub repository, issue tracker or other online assets. You can also enter markdown text in the *Edit resources* tab to include additional information helpful to end users who may use your collection in their automation tasks.
+You can add information and provide resources for your users to accompany collections included in the namespace. Add a logo, a description and link users to your GitHub repository, issue tracker, or other online assets. You can also enter markdown text in the *Edit resources* tab to include more information. This is helpful to end users who may use your collection in their automation tasks.
```

Comment:
The syntax here is a little confusing. Would 
'Add a logo and a description, and link users to your GitHub repository, issue tracker, or other online assets.' work?

Reply:
Hey Lizzi, thanks for reviewing. I agree with the syntax suggested and updated. Thanks!